### ISSUE TITLE
typos in the Berkeley growth dataset

### ISSUE THREAD

**Issue by babeheim** [statist7/sitar#7]
My team at MPI-EVA recently transcribed the data from Tuddenham, Read D. and Snyder, Margaret M. (1954), aka `berkeley.rda`, before realizing it was already done in this package! Comparing the two transcriptions, we spotted a few small typos which are worth correcting.

| id    |   age    |   variable   |   sitar   |   book  |
|-------|----------|--------------|-----------|---------|
| 207   |  0.25    |   weight     |   8.5     |   6.5   |
| 208   |  1.75    |   weight     |   13.5     |   13.9   |
| 221   |    12    |   weight     |   41.3     |   47.3   |
| 227   |    3     |   weight     |   14.1     |   14.7   |
| 258   |    3     |   weight     |   16.1     |   16.7   |
| 269   |    2     |   weight     |   11.2     |   12.2   |
| 274   | 11.5     |  weight      | 31.0       |     37.0  |
| 317   | 18.0     |  weight      | 54.1       |     54.7  |
| 364   | 13.0     |  weight      | 35.5       |     35.9  |
| 369   | 11.0     |  weight      | 35.0       |     39.0  |

And these for height:

| id    |   age    |   variable   |   sitar   |   book  |
|-------|----------|--------------|-----------|---------|
| 203  |  15.50  |  height |   165.7  |    168.7 |
| 225  |  12.50  |  height |   154.1  |    154.4 |
| 238  |   0.25  |  height |    60.3  |     60.8 |
| 276  |   0.75  |  height |    72.1  |     72.4 |
| 277  |   1.50  |  height |    82.1  |     82.4 |
| 307  |   1.75  |  height |    85.0  |     88.0 |
| 325  |   0.75  |  height |    67.4  |     67.1 |
| 325  |  18.50  |  height |   156.1  |    156.4 |
| 383  |  18.50  |  height |   162.5  |    162.8 |

We are not including the following "errors", which are clearly corrections on your part from what must have been mistakes when they first prepared these tables in 1954.

| id    |   age    |   variable   |   sitar   |   book  |
|-------|----------|--------------|-----------|---------|
| 201   |   16.50  |  height      |   177.2    |    172.2  |
| 304   |    5.00  |  height      |   113.9    |    103.9  |
| 325   |    3.00  |  height      |    88.8    |     83.8  |

**Comment by babeheim** [comment]
We can provide scanned PDFs of each page, if you want to verify our numbers!

**Comment by statist7** [comment]
Thanks very much! 

I'll be pleased to incorporate your changes into the package, where the errors were presumably generated during the scanning process.

I assume too that the "errors" that I fixed are present in the book - is that right?

**Comment by babeheim** [comment]
> I assume too that the "errors" that I fixed are present in the book - is that right?

Yes, that's right. One thing also to note is that the book contains a certain amount of imputed values - these are indicated by underlines and parenthesis around the values. These data were originally missing but filled in by averaging surrounding known values in the trajectory. Our colleague John Bunce here at MPI-EVA is currently working on an analysis of this data and needed that information coded.

**Comment by statist7** [comment]
Thanks, that's useful.

It would be good to be able to offer the raw (i.e. unimputed) dataset as an alternative. Will that be an outcome from John Bunce's analysis, and if so, would it be available?
